src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -703,7 +760,6 @@ public String getStatus() {
         PUT
     }
 
-    // represents Errors returned as JSON
```

Comment:
Why was this removed? Why was it here to begin with?

Reply:
I can re-add it if needed. I believe it was just an explainer comment on what the `Error` class was for.